baseplate/thrift.go

```diff
@@ -36,13 +37,14 @@ func (bts *baseplateThriftServer) Serve() error {
 }
 
 func (bts *baseplateThriftServer) Close() error {
-	var err error
-	bts.thriftServer.Stop()
+	errors := &batcherror.BatchError{}
```

Comment:
nit: BatchError supports both reference and pointer, so `var errors batcherror.BatchError` is sufficient (but this still works)